openstf/model/xgb_quantile.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from typing import Tuple
+from functools import partial
+
+from sklearn.base import RegressorMixin, BaseEstimator
+from sklearn.utils.validation import check_X_y, check_array, check_is_fitted
+from xgboost import XGBRegressor
+import numpy as np
+
+import openstf.metrics.metrics as metrics
+
+DEFAULT_QUANTILES: Tuple[float, ...] = (0.9, 0.5, 0.1)
+
+
+class XgbQuantile(BaseEstimator, RegressorMixin):
+    def __init__(self, quantiles: Tuple[float, ...] = DEFAULT_QUANTILES):
+
```

Comment:
I think we should initialize all attributes in the constructor for more clarity. So in this case `estimators_` and `is_fitted_` or are those `RegressorMixin` attributes?

Reply:
These are requirements by sklearn. If you add these parameters to the __init__ the compatibility check will not pass. The "_" postfix is the sklearn way of defining private attributes.